MITADS/opensubtitles_exporter.py

```diff
@@ -121,22 +122,19 @@ def parsexmlfile(xml_path, count_file):
 count_file = 0
 
 # Parse 5 files at once
-pool = ThreadPoolExecutor(max_workers=20)
+pool = ProcessPoolExecutor(max_workers=20)
 
-total_lines = 0
-for xml_path in pathlist:
+
+paths = []
```

Comment:
Nit: it doesn't matter too much, but you could put the for loop in a generator function and call map with it rather than collecting the paths in a list first.